lib/twitter_cldr/js/renderers/numbers/rbnf/rbnf_renderer.rb

```diff
@@ -0,0 +1,24 @@
+# encoding: UTF-8
+
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+module TwitterCldr
+  module Js
+    module Renderers
+      module Numbers
+
+        class RBNFRenderer < TwitterCldr::Js::Renderers::Base
+          set_template "mustache/numbers/rbnf/rbnf.coffee"
+
+          def resource
+            result = {}
```

Comment:
Whenever you have to initialize an empty container like that in Ruby, something is not right :) Here you can just do

```ruby
{
  locale => ....
}.to_json
```